lib/alias.ps1

```diff
@@ -0,0 +1,83 @@
+. "$PSScriptRoot\..\lib\install.ps1" # shim related
+
+$script:config_alias = 'alias'
+
+function init_alias_config {
+    $aliases = get_config $script:config_alias
+    if ($aliases) {
+        $aliases
+    } else {
+        New-Object -TypeName PSObject
```

Comment:
Just create an object and doesn't assign it?

Reply:
Same as above. The necessary refactoring will be done.